voting/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
as I said in prev PR - license should live on the outside and should probably copy one we use everywhere else.